examples/campaign_management/add_campaign_labels.py

```diff
@@ -0,0 +1,97 @@
+    """This code example adds a campaign label to a list of campaigns.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: A client customer ID str.
+        label_id: A label ID to use when searching for campaigns.
+        campaign_ids: A list of campaign IDs to which the label will be added.
+    """
+
+    # Get an instance of CampaignLabelService client.
+    campaign_label_service = client.get_service(
+        'CampaignLabelService', version='v3')
+    campaign_service = client.get_service('CampaignService', version='v3')
+    label_service = client.get_service('LabelService', version='v3')
+
+    # Set the resource name of the label to be added across the campaigns.
+    label_resource_name = label_service.label_path(customer_id, label_id)
+
+    operations = []
+
+    for campaign_id in campaign_ids:
+        campaign_resource_name = campaign_service.campaign_path(customer_id,
+                                                                campaign_id)
+        campaign_label = client.get_type('CampaignLabel', version='v3')
+        campaign_label_operation = client.get_type(
+            'CampaignLabelOperation', version='v3')
+
+        campaign_label = campaign_label_operation.create
+        campaign_label.campaign.value = campaign_resource_name
+        campaign_label.label.value = label_resource_name
+        operations.append(campaign_label_operation)
+
+    try:
+        response = campaign_label_service.mutate_campaign_labels(
+            customer_id, operations)
+        print(f'Added {len(response.results)} campaign labels:')
+        for result in response.results:
+            print(result.resource_name)
+    except GoogleAdsException as error:
+        print('Request with ID "{}" failed with status "{}" and includes the '
+              'following errors:'.format(
+                  error.request_id, error.error.code().name))
+        for error in error.failure.errors:
+            print('\tError with message "{}".'.format(error.message))
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: {}'.format(
+                        field_path_element.field_name))
+        sys.exit(1)
+
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description='Lists all campaigns for specified customer.')
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument('-c', '--customer_id', type=str,
+                        required=True, help='The Google Ads customer ID.')
+    parser.add_argument('-l', '--label_id', type=str, required=True,
+                        help='A label ID associated with a campaign.')
```

Comment:
The descriptor  of this parameter seems wrong to me, in PHP we use "the ID of the label to attach to campaigns"

Reply:
Fixed.